templates/default/sentry.conf.8.0.py.erb

```diff
@@ -0,0 +1,205 @@
+# This file is just Python, with a touch of Django which means
```

Comment:
you can run `diff -u sentry.conf.py.erb sentry.conf.8.0.py.erb` (or use colordiff on your mac)